packages/vite/src/utils.ts

```diff
@@ -1,3 +1,21 @@
+import { getPaths } from '@redwoodjs/project-config'
+
 export function stripQueryStringAndHashFromPath(url: string) {
   return url.split('?')[0].split('#')[0]
 }
+
+// Check CWD: make sure its the web/ directory
+// Without this Postcss can misbehave, and its hard to trace why.
+export function ensureProcessDirWeb(webDir: string = getPaths().web.base) {
+  if (process.cwd() !== webDir) {
+    console.error('⚠️  Warning: CWD is ', process.cwd())
+    console.warn('~'.repeat(50))
+    console.warn(
+      'The cwd must be web/. Please use invoke with yarn rw {command} or run the command from the web/ directory.'
```

Comment:
Probably meant "use" or "invoke" here; and doesn't matter much but yargs uses `<>` to signal required positionals so users may be more familiar with that instead of `{}` which is more React-y:

```suggestion
      'The cwd must be web/. Please use `yarn rw <command>` or run the command from the web/ directory.'
```